characters/anemo/kazuha.md

```diff
@@ -72,6 +72,7 @@ Unleashes a secret technique as fierce as the rushing wind that pulls objects an
 * **Press:** Can be used in mid-air.
 
 * **Hold:** Charges up before unleashing greater Anemo DMG over a larger AoE than Press Mode. 
+  * Kazuha Hodling version of **Chihayaburu** can cause certain types of Enemies to take Fall DMG.
```

Comment:
Holding

Reply:
actually wowrding in general is awkward, probably just say something along the lines of 
"This version/variant can cause certain enemies to take Fall Damage"

Reply:
fixed